lib/orbit_web/controllers/sign_in_export_controller.ex

```diff
@@ -31,6 +31,18 @@ defmodule OrbitWeb.SignInExportController do
     end
   end
 
+  @spec override_expiration_date([map()], String.t()) :: String.t()
+  def override_expiration_date(override_list, type) do
+    {:ok, datetime, _} =
+      Enum.find(override_list, fn cert -> cert["type"] == type end)["expires"]
```

Comment:
bug: This will crash if given a type that's not in the list. But below, if there are any overrides, then this is called with both types, even if only one type was overridden.

This should gracefully return `""` or `nil` if the type isn't found.

Reply:
[fixed](https://github.com/mbta/orbit/pull/220/files#diff-10e30af09b064503abd15c28480b0de62aaab8694e92cb3af201ae79a85662d1R36), thanks! using "" as it needs to CSV happy.